scrapers_next/in/people.py

```diff
@@ -0,0 +1,359 @@
+
+        addr = CSS("div.sen-contact p").match(self.root)[0].text_content()
+
+        # no phone for this link
+        if self.source.url == "https://www.indianasenaterepublicans.com/young":
+            addr = addr
+            phone1 = None
+            phone2 = None
+        else:
+            addr, phone1, phone2 = re.search(
+                r"(.+)Phone:\s(\d{3}-\d{3}-\d{4})\s?or\s(\d{3}-\d{3}-\d{4})", addr
+            ).groups()
+
+        p.capitol_office.address = addr
+        if phone1:
+            p.capitol_office.voice = phone1
+        if phone2:
+            p.extras["second phone"] = phone2
+
+        if len(CSS("div.sen-contact p").match(self.root)) == 1:
+            leg_assist = CSS("div.sen-contact p").match_one(self.root).text_content()
+        else:
+            leg_assist = CSS("div.sen-contact p").match(self.root)[1].text_content()
+
+        if len(CSS("div.sen-contact p").match(self.root)) < 3:
+            extra_contacts = leg_assist.split("Media Contact:")
+            leg_assist = extra_contacts[0]
+            media_contact = extra_contacts[1]
+            leg_assist_name, leg_assist_phone, leg_assist_email = re.search(
+                r"Legislative\sAssistant:?(.+)Phone:\s(.+)Email:\s(.+)", leg_assist
+            ).groups()
+            media_contact_name, media_contact_phone, media_contact_email = re.search(
+                r"(.+)Phone:\s(.+)Email:\s(.+)", media_contact
+            ).groups()
+        elif (
+            len(CSS("div.sen-contact p").match(self.root)) == 3
+            or self.source.url == "https://www.indianasenaterepublicans.com/bray"
+        ):
+            leg_assist_name, leg_assist_phone, leg_assist_email = re.search(
+                r"Legislative\sAssistant:?(.+)Phone:\s(.+)Email:\s(.+)", leg_assist
+            ).groups()
+            media_contact = CSS("div.sen-contact p").match(self.root)[2].text_content()
+            media_contact_name, media_contact_phone, media_contact_email = re.search(
+                r"Media\sContact:(.+)Phone:\s(.+)Email:\s(.+)", media_contact
+            ).groups()
+        else:
+            leg_assist_name, leg_assist_phone, leg_assist_email = re.search(
+                r"Legislative\sAssistant:?(.+)Phone:\s(.+)Email:\s(.+)", leg_assist
+            ).groups()
+            media_contact = CSS("div.sen-contact p").match(self.root)[3].text_content()
+            media_contact_name, media_contact_phone, media_contact_email = re.search(
+                r"Media\sContact:(.+)Phone:\s(.+)Email:\s(.+)", media_contact
+            ).groups()
+
+        p.extras["legislative assistant name"] = leg_assist_name
+        p.extras["legislative assistant phone"] = leg_assist_phone
+        p.extras["legislative assistant email"] = leg_assist_email
+        p.extras["media contact name"] = media_contact_name
+        p.extras["media contact phone"] = media_contact_phone
+        p.extras["media contact email"] = media_contact_email
+
+        """
+        try:
```

Comment:
Could potentially grab education info here, but it gets messy for some people with multi-lines.